xAPI.md

```diff
@@ -1862,6 +1862,10 @@ The Experience API applies the concept of registration more broadly.  A registra
 considered to be an attempt, a session, or could span multiple Activities. There is no expectation that 
 completing an Activity ends a registration. Nor is a registration necessarily confined to a single Agent.
 
+The Registration id is also used when storing documents within the State resource, e.g. for 
```

Comment:
I'm not keen on adding the SHOULD requirement because I can't see any interoperability reason for doing that, it's just a sensible thing to do. For that reason I prefer the non-requirement "normally..." description.